src/core/services/UserService.ts

```diff
@@ -1,17 +1,101 @@
-import { GetAddressEnsNameProps, UserService, Web3Provider } from '@types';
+import { GetAddressEnsNameProps, UserService, Web3Provider, NftBalances, Config } from '@types';
+import { getContract } from '@frameworks/ethers';
+
+import bluePillNFTAbi from './contracts/bluePillNFT.json';
+import woofyNFTAbi from './contracts/woofyNFT.json';
+import BigNumber from 'bignumber.js';
+import { Contract } from 'ethers';
```

Comment:
We have not documented a convention, but try moving external dependencies to top.